geoh5py/ui_json/input_file.py

```diff
@@ -270,16 +271,58 @@ def update_ui_values(self, data: dict):
             else:
                 self.ui_json[key] = value
 
+    @property
+    def validate(self):
+        """Option to run validations."""
+        return self._validate
+
+    @validate.setter
+    def validate(self, value: bool):
+        if not isinstance(value, bool):
+            raise ValueError("Input value for `validate` should be True or False.")
+
+        self._validate = value
+
     @property
     def validation_options(self):
-        """Pass validation options to the validators."""
+        """
+        Pass validation options to the validators. The following parameters are
```

Comment:
Split the lines.  Should be a one liner followed by a new paragraph for details like the supported params

Reply:
Done